src/js/utils/graphics.js

```diff
@@ -36,4 +36,6 @@ export const arcCommands = (centerX, centerY, radius, startAngle, endAngle) => {
 };
 
 export const translateEndAngle = (startAngle, anglePer, value) =>
-  Math.min(360, Math.max(0, startAngle + anglePer * value));
+  Math.max(0, startAngle + anglePer * value) % 360;
+// Math.min(360, Math.max(0, startAngle + anglePer * value));
```

Comment:
Remove unused code. Also can you add a comment around what this is doing since we're changing the logic? will be helpful for future

Reply:
aw yes, good catch! Didn't realize that was still in there